staff/maryam-malek/app-template/profile.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Profile</title>
+</head>
+<body>
+    <header>
+        <h2>Profile</h2>
+    </header>
+    <section>
+        <h4>Name </h4>
+        <p></p>
+    </section>
+    <section>
+        <h4>Surname</h4>
+        <p></p>
+    </section>
+    <section>
+        <h4>Email</h4>
+        <p></p>
+    </section>
```

Comment:
may you need to change that three above sections for ```p```s (or whatever) inside an address tag